source/includes/steps-realm-cli-generate-api-key.yaml

```
@@ -22,12 +22,17 @@ content: |
      :alt: Click Access Manager
      :lightbox:
 
-  Clicking this button will navigate you to the "Create API Key" screen.
-  Set a description for your key and a project permission. The various
-  project permissions are described on the right hand side of the screen
-  and can be changed at a later time. Copy the public key to use later
-  in order to log in. Click :guilabel:`next` to continue configuring your
-  key details.
+  Clicking this button will navigate you to the "Create API Key" screen. Set a
+  description for your key.
+
+  For write access, the CLI requires an API key with Project Owner-level
+  permissions. Use the :guilabel:`Project Permissions` dropdown to select
+  "Project Owner".
+
+  If you only need read access, you can use "Project Read Only".
```

It feels a little strange to frontload write permissions here, describe how to select the write permission, then mention the read-only permission as an afterthought. How about:

```suggestion
  For write access, the CLI requires an API key with Project Owner-level
  permissions. If you only need read access, you can use "Project Read Only".
  Use the :guilabel:`Project Permissions` dropdown to select a permission level.
```